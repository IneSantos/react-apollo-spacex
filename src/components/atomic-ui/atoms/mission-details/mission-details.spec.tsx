import React from "react";
import { render } from "@testing-library/react";
import MisisonDetails from "./mission-details";

describe("MisisonDetails atom tests", () => {
  test(`MisisonDetails component render`, () => {
    const { container } = render(<MisisonDetails details="This is a detail" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
