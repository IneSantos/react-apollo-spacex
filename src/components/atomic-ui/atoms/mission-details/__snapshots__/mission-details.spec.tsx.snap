// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MisisonDetails atom tests MisisonDetails component render 1`] = `
<div
  class="details"
>
  This is a detail
</div>
`;
